{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/commands.py

```diff
@@ -12,11 +12,21 @@
 
 
 @click.command()
-def test():
+@click.option(
+    "-c",
+    "--coverage",
+    default=False,
+    is_flag=True,
+    help="Show coverage report",
+)
+def test(coverage):
     """Run the tests."""
     import pytest
 
-    rv = pytest.main([TEST_PATH, "--verbose"])
+    args = [TEST_PATH, "--verbose"]
+    if coverage:
+        args = ["--cov={}".format(HERE), "--verbose"]
```

Comment:
Instead of overriding the list of args in the `coverage` block, what about setting the default args on line 26 and then append optional args below?

Reply:
sure, converted to append